packages/components/Molecules/Card/CardNavigationV2.vue

```diff
@@ -0,0 +1,346 @@
+<template>
+  <rpl-link v-if="link" :class="modifiers" :href="link.url" :innerWrap="false">
+    <div v-if="image" class="rpl-card-nav__image-wrapper">
+      <rpl-responsive-img class="rpl-card-nav__image" v-bind="image" alt="" :srcSet="srcSet" />
+    </div>
+    <div class="rpl-card-nav__content">
+      <div v-if="formattedDate || tag || status" class="rpl-card-nav__meta">
+        <div v-if="tag" class="rpl-card-nav__tag" >{{ tag }}</div>
+        <div v-if="status" class="rpl-card-nav__status" :class="`rpl-card-nav__status--${this.status.toLowerCase()}`">
+          <rpl-icon v-if="statusIcon" class="rpl-card-nav__status-icon" :symbol="statusIcon.symbol" :color="statusIcon.color" size="s" />
+          <span>{{ status }}</span>
+        </div>
+        <div v-if="formattedDate" class="rpl-card-nav__date">{{ formattedDate }}</div>
+      </div>
+      <h2 v-if="title" class="rpl-card-nav__title"><span>{{ trimmedTitle }}</span></h2>
+      <p v-if="summary" class="rpl-card-nav__summary">{{ trimmedSummary }}</p>
+      <p v-if="author" class="rpl-card-nav__author"><strong>Author:</strong> {{ author }}</p>
+    </div>
+  </rpl-link>
+</template>
+
+<script>
+import RplLink from '@dpc-sdp/ripple-link'
+import RplIcon from '@dpc-sdp/ripple-icon'
+import RplResponsiveImg from '@dpc-sdp/ripple-responsive-img'
+import formatdate from '@dpc-sdp/ripple-global/mixins/formatdate'
+import card from '@dpc-sdp/ripple-card/mixins/card'
+
+export default {
+  name: 'RplCardNavigationV2',
```

Comment:
Dylan didn't use V2 in name for his new promo, do you want to do the same?

Reply:
I actually prefer having no V2 (we don't need to rename as well when we remove unused cards). I'll update this to `RplCardNav` so it aligns with the shorten RplCardPromo (: